.gitignore

```diff
@@ -1,3 +1,2 @@
-# Dir for bundles
-dist
```

Comment:
Why did you remove it? This is a build folder right?

Reply:
I updated the package config to generate the openapi file in the root folder (just like the old, hand-written one). So the dist folder is not necessary.